styles/google/catppuccin.user.css

```diff
@@ -0,0 +1,1085 @@
+    }
+    .c07z9 {
+      color: @subtext0;
+    }
+    .c07z9 a,
+    .c07z9 a:active,
+    .c07z9 a:link {
+      color: @text;
+      text-decoration-color: @subtext0;
+    }
+    .wHYlTd {
+      color: @subtext0;
+    }
+    .dG2XIf .c2xzTb .wDYxhc:first-child {
+      background: @base;
+    }
+    .fm06If .ILfuVd,
+    .c2xzTb .ILfuVd,
+    .Jb0Zif .c2xzTb .ILfuVd {
+      color: @text;
+    }
+    .fm06If .ILfuVd b,
+    .c2xzTb .ILfuVd b {
+      background-color: fade(@blue, 25%);
+      color: @text;
+    }
+    .GKS7s {
+      background: @surface0;
+      border-color: @surface1;
+    }
+    .GKS7s:hover {
+      background-color: @surface1 !important;
+      border-color: @surface2;
+    }
+    .yXK7lf em {
+      color: @text;
+    }
+    .Lb0dA:hover {
+      background-color: @surface0;
+    }
+    .Lb0dA[aria-expanded="true"]:hover .EGarzf {
+      color: @blue;
+    }
+    .Lb0dA[aria-expanded="true"] {
+      background-color: fade(@blue, 25%);
+    }
+    .BaegVc:hover {
+      background-color: @surface0;
+    }
+    .hdtb-tl-sel .BaegVc {
+      background-color: fade(@blue, 25%);
+    }
+    .BaegVc:focus {
+      background-color: fade(@blue, 25%);
+    }
+    .TZqsAd {
+      background: @overlay0;
+    }
+    .FalWJb {
+      background: @base;
+    }
+    .YeThId {
+      border-top-color: @surface1;
+    }
+    .k8XOCe {
+      background-color: @surface0;
+    }
+    .s75CSd {
+      color: @text;
+    }
+    .ZkkK1e {
+      background: @base;
+      border-color: @surface1;
+    }
+    .ZkkK1e:hover {
+      background: @surface0;
+    }
+    .QjI2jc {
+      color: @subtext0;
+    }
+    .lhLbod {
+      color: @subtext0;
+    }
+    .nr7I6e div[aria-expanded="true"] .F75bid,
+    .nr7I6e div[aria-expanded="true"]:hover .fFI3rb .F75bid {
+      color: @blue;
+    }
+    .F75bid {
+      color: @subtext0;
+    }
+    div[aria-expanded="true"] .fFI3rb {
+      background-color: fade(@blue, 25%) !important;
+    }
+    div[aria-expanded="true"]:hover .fFI3rb:hover {
+      background-color: fade(@blue, 25%) !important;
+    }
+    .MAMMm {
+      background: @mantle;
+      box-shadow: 0 2px 6px @crust !important;
+    }
+    [selected].nWtjMe {
+      background: fade(@blue, 25%) !important;
+    }
+    .uZNuIf:hover {
+      background-color: @surface0;
+      color: @text;
+    }
+    [selected].nWtjMe .FMKtTb {
+      color: @text;
+    }
+    .nWtjMe .FMKtTb {
+      color: @subtext0;
+    }
+    .GKS7s .FMKtTb,
+    .GKS7s .RWhoyd.mol54e {
+      color: @text;
+    }
+    .ePSouf .FMKtTb {
+      color: @subtext0;
+    }
+    .NkCsjc:hover {
+      background-color: @surface0 !important;
+      color: @text;
+    }
+    .nr7I6e div[aria-haspopup="true"]:hover .F75bid {
+      color: @text;
+    }
+    a.hisnlb {
+      background: fade(@overlay2, 25%);
+    }
+    div[aria-expanded="true"] .fFI3rb,
+    div[aria-expanded="true"]:hover .fFI3rb:hover {
+      background-color: @surface0 !important;
+    }
```

Comment:
While this sets it to `surface0`.

This rule should currently be applied, I think, since it's lower in the CSS file.

Reply:
I remember the other 'blue' 25% fade being more important